outplacement_crmsync/models/res_partner.py

```diff
@@ -50,17 +69,143 @@ def get_jobseeker_data(self):
             'res.partner', 'search_read',
                 [
                     # ~ [['social_sec_nr', '=', self.social_sec_nr]],
-                    [[]],
+                    [['id','=',26]],
```

Comment:
should this be hardcoded? What if there is no partner with id = 26?

Reply:
yes, i like hard coded test data ;-)